runway/module/kustomize.py

```diff
@@ -0,0 +1,123 @@
+"""Kustomize module."""
+
+import logging
+import os
+import subprocess
+import sys
+
+from . import RunwayModule, warn_on_skipped_configs
+from ..util import which
+
+LOGGER = logging.getLogger('runway')
+
+
+def get_kustoimze_config_file(path, environment):
+    """Return kustomze environment file name."""
```

Comment:
typo kustomze > kustomize ?

Reply:
This was just a test to see if you were paying attention :-D

Reply:
Totally 🤣